client/src/Command.ts

```diff
@@ -66,7 +66,7 @@ export function extractTextStyles(Command: BaseCommand): TextStyles {
       return style;
 
     case "SelectOptionCommand":
-      return new TextStyles();
+      return new TextStyles("Unstyled Option");
```

Comment:
What does this mean?

Reply:
Options don't carry styles from server, but they need to fill the StyleArray to sync with everything else, so they carry a marked empty style. Just allows us to be explicit about what kind of TextStyle we've received.

Reply:
Ok…